content/docs/install/multi-cluster/index.md

```diff
@@ -163,6 +163,13 @@ The important once that are used in the above example are:
 | `remoteControlPlane.api.token` | Keptn API token | `""` |
 
 
+## Known Limitations
+
+Keptn uses the integration name and version as identifier for connecting the integration to the control plane.
+Hence, in the case of multiple remote execution planes, each integration must have a different name/version pair, otherwise Keptn will assign the same set of subscriptions.
+To avoid this, we recommend to assign a different name to the integrations.
```

Comment:
```suggestion
To avoid this,  you must assign a different name to each integration.
```
I'm thinking that this is not really optional -- they just have to do it, right?